controllers/staff.js

```diff
@@ -185,7 +185,43 @@ class staffController{
         }catch(error){
             next(error)
         }
-    }  
+    }
+    
+    static async createBannedPost(req, res, next) {
+        const transaction = await sequelize.transaction();
+        try {
+            const {userType} = req.decodedToken;
+            if(userType != 'staff'){
+                throw ({name: "UNAUTHORIZED"});
+            }
+            const { username } = req.decodedToken;
```

Comment:
De-structure, ur variable once. On line 193, you can declare a username variable there.